config/samples/loggingplumber_v1alpha1_flowtest.yaml

```diff
@@ -2,6 +2,7 @@ apiVersion: loggingplumber.isala.me/v1alpha1
 kind: FlowTest
 metadata:
   name: flowtest-sample
+  namespace: cert-manager
```

Comment:
why is this in cert manager?

Reply:
oh I forgot to change this I did it to see UI show the namespace correctly